modules/kernel/src/main/scala/notebook/Messages.scala

```diff
@@ -18,6 +18,8 @@ sealed trait CalcResponse
 
 case class StreamResponse(data: String, name: String) extends CalcResponse
 
+case class DefinitionResponse(definedTermOrType: Option[Either[String, String]], tpe:String, cell:String, references: List[String]) extends CalcResponse
```

Comment:
I kindo dislike this either, better using an ADT one case class for Term and Type